### ISSUE TITLE
Embedding did as a Teams-app

### ISSUE THREAD

**Issue by damsleth** [Puzzlepart/did#794]
### TLDR
How can we make this happen?
![image](https://user-images.githubusercontent.com/7300548/105462520-03471180-5c8f-11eb-8c9f-9f3534da12f6.png)

### Discussion
See [this thread](https://teams.microsoft.com/l/message/19:876c3ec2e97e4186b71cbd01f5002963@thread.tacv2/1611301463889?tenantId=dcb89c10-67b8-48cc-8819-85de37e8af91&groupId=46adb6c8-8c09-409f-a333-2d131ab6aff8&parentMessageId=1611301463889&teamName=Did&channelName=General&createdTime=1611301463889) and [this thread](https://teams.microsoft.com/l/message/19:1eae41833c42443cb410524b28d7151b@thread.tacv2/1611235704784?tenantId=dcb89c10-67b8-48cc-8819-85de37e8af91&groupId=8f4e700a-62e0-41a7-9f53-e8c8bd4a0427&parentMessageId=1611235704784&teamName=NyeMoss&channelName=General&createdTime=1611235704784).  

![image](https://user-images.githubusercontent.com/7300548/105463340-363dd500-5c90-11eb-8a67-9694b825aee9.png)

Offering did as an add-on feature with Teams might be a big opportunity

### Current hurdles
Authentication won't work, since Teams-apps are embedded in an iframe, and login.microsoftonline.com has `X-frame-Options` set to `deny`. **This is a blocker.**.
